ui/admin/app/components/auth-methods/auth-method/actions/index.hbs

```diff
@@ -40,7 +40,7 @@
   </Rose::Dropdown>
 {{/if}}
 
-<Hds::Dropdown data-test-manage-auth-methods-dropdown as |dd|>
+<Hds::Dropdown data-test-manage-auth-method as |dd|>
```

Comment:
This `data-test` attribute didn't exactly match so I removed the `s` since this is for a single auth-method.
I also removed the `dropdown`. We actually have a mix of how we name these attributes. Might be something we discuss during a tech time.

Reply:
i think removing `dropdown` wording is reasonable.